frontend/packages/client/src/components/StepByStep/index.js

```diff
@@ -1,9 +1,37 @@
 import React, { useState, useCallback } from "react";
```

Comment:
This component is very powerful! We will have to add some docs or explanatory tests as the codebase continues to grow.